src/components/seeds/database/index.tsx

```diff
@@ -407,33 +410,111 @@ export default function Database(props: DatabaseProps): JSX.Element {
     }
   };
 
+  const onSeedBankForImportSelected = (selectedFacility: Facility | undefined) => {
+    setSelectSeedBankForImportModalOpen(false);
+    if (selectedFacility) {
+      setSelectedOrgInfo({ ...selectedOrgInfo, selectedFacility });
+      setOpenImportModal(true);
+    }
+  };
+
+  const importAccessions = () => {
+    setSelectSeedBankForImportModalOpen(true);
+  };
+
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null);
+  const open = Boolean(anchorEl);
+  const id = open ? 'simple-popover' : undefined;
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
   const getHeaderButtons = () => (
     <>
-      <Button
-        id='edit-columns'
-        label={strings.ADD_COLUMNS}
-        onClick={onOpenEditColumnsModal}
-        priority='secondary'
-        type='passive'
-        size='medium'
-        className={classes.buttonSpc}
-      />
-      <Button
-        id='download-report'
-        label={strings.DOWNLOAD_AS_REPORT}
-        onClick={onDownloadReport}
-        priority='secondary'
-        type='passive'
-        size='medium'
-        className={classes.buttonSpc}
-      />
+      <ButtonMUI
+        aria-describedby={id}
+        variant='contained'
+        onClick={handleClick}
+        sx={{
+          borderRadius: '40px',
+          width: '40px',
+          minWidth: '40px',
+          color: '#0067C8',
+          background: '#fff',
+          border: '2px solid #0067C8',
+          padding: 0,
+          fontSize: '20px',
+          marginRight: 1,
+        }}
+      >
+        ...
+      </ButtonMUI>
+      <Popover
+        id={id}
+        open={open}
+        anchorEl={anchorEl}
+        onClose={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'left',
+        }}
+      >
+        <Box>
```

Comment:
looks a bit strange? 
<img width="189" alt="Terraware App 2022-09-26 08-02-29" src="https://user-images.githubusercontent.com/1865174/192312288-81e48191-0d41-4838-a92d-413b4b5f7cc3.png">

Reply:
Yes, I just moved the two buttons inside the menu, but how should it look? Like a simple menu?

Reply:
<img width="280" alt="SBD_V2 – Figma 2022-09-26 08-25-13" src="https://user-images.githubusercontent.com/1865174/192317025-1e7a365a-4f20-46fe-abdb-f6a9a46363ae.png">

Reply:
Thanks!